.github/workflows/build.yml

```
@@ -18,20 +18,17 @@ jobs:
         strategy:
             fail-fast: false
             matrix:
-                php: [7.4, 8.0]
+                php: [8.0, 8.1]
                 node: [10.x]
                 mysql: [5.7, 8.0]
                 symfony: [^4.4, ~5.2.0, ~5.3.0]
-                sylius: [~1.9.0, ~1.10.0]
-                exclude:
-                    - # Sylius 1.9 does not support PHP 8.0
-                        sylius: ~1.9.0
-                        php: 8.0
+                sylius: [~1.10.0]
```

Why didn't you add 1.11?
```suggestion
                sylius: [~1.10.0, ~1.11.0]
```